src/lib/mongoBackend/mongoUpdateContextSubscription.cpp

```diff
@@ -245,8 +245,36 @@ HttpStatusCode mongoUpdateContextSubscription
       /* Build conditions array (including side-effect notifications and threads creation)
        * In order to do so, we have to create and EntityIdVector and AttributeList from sub
        * document, given the processConditionVector() signature */
-       EntityIdVector enV = subToEntityIdVector(sub);
-       AttributeList attrL = subToAttributeList(sub);
+       EntityIdVector enV;
+       AttributeList attrL;
+       if (version == "v1")
+       {
+         enV = subToEntityIdVector(sub);
+         attrL = subToAttributeList(sub);
+       }
+       else // v2
+       {
+         // In v2 entities and attribute are updatable (as part of subject or notification) wo
+         // we have to check in order to know if we get the attribute from the request or from
+         // the subscription
+         if (requestP->entityIdVector.size() > 0)
+         {
+           enV = requestP->entityIdVector;
```

Comment:
Double-free potential problem? I mean, enV releasing and requestP releasing end releasing the same piece of allocated dynamic memory.

Let's see how test harness behave regarding this (double-free will cause segfault in some causes, so some tests should faild).